dist/preview release/what's new.md

```diff
@@ -14,6 +14,7 @@
 - Added predicate function to customize the list of mesh included in the computation of bounding vectors in the ```getHierarchyBoundingVectors``` method ([sebavan](http://www.github.com/sebavan))
 - Added webVR constructor options: disable laser pointer toggle, teleportation floor meshes ([TrevorDev](https://github.com/TrevorDev))
 
+

```

Comment:
Please add a line in the major update to present your great work?

Reply:
Oh I will do that :)